test/e2e/tests/test_db_instance.py

```diff
@@ -97,14 +97,14 @@ def postgres14_t3_micro_instance(k8s_secret):
     # Try to delete, if doesn't already exist
     try:
         _, deleted = k8s.delete_custom_resource(ref, 3, 10)
-        assert deleted
-        db_instance.wait_until_deleted(db_instance_id)
     except:
         pass
+    db_instance.wait_until_deleted(db_instance_id)
+
 
 @service_marker
 @pytest.mark.canary
-class TestDBInstance:  
+class TestDBInstance:
```

Comment:
Can we uncomment the tests below in `L164` and `L180`?

Reply:
> Can we uncomment the tests below in `L164` and `L180`?

good catch. yeah. I will update.